src/main/java/org/takes/tk/TkCors.java

```diff
@@ -34,6 +34,7 @@
 import org.takes.Response;
 import org.takes.Take;
 import org.takes.rq.RqHeaders;
+import org.takes.rs.RsPrettyJson;
```

Comment:
@xupyprmv not sure if we should do a import just to mention class in a puzzle :)